src/js/authEthereum.js

```diff
@@ -69,6 +69,13 @@ async function login () {
 }
 
 onClick('authEthereum', login)
+onClick('switchEthWallet', async () => {
+  window.ethInitialized = false
+  window.dom.hide('unsupportedNetworkModal')
```

Comment:
Should probably move this to the end of the function, even if in practice the wait is never long enough to cause confusion